server/services/topics.js

```diff
@@ -1,24 +1,17 @@
 const caseInsensitive = new Intl.Collator('en', { caseFirst: 'lower' });
 
 class TopicsService {
-  #hubMenuRepository;
-
   #cmsApi;
 
-  constructor(hubMenuRepository, cmsApi) {
-    this.#hubMenuRepository = hubMenuRepository;
+  constructor(cmsApi) {
     this.#cmsApi = cmsApi;
   }
 
   async getTopics(prisonId) {
-    const [tags, primary] = await Promise.all([
-      this.#hubMenuRepository.tagsMenu(prisonId),
-      this.#cmsApi.primaryMenu(prisonId),
-    ]);
-
-    return tags
-      .concat(primary)
-      .sort((a, b) => caseInsensitive.compare(a.linkText, b.linkText));
+    const topics = await this.#cmsApi.getTopics(prisonId);
+    return topics.sort((a, b) =>
+      caseInsensitive.compare(a.linkText, b.linkText),
+    );
```

Comment:
could potentially sort in the api layer?

Reply:
Yeah I think ideally sorting would be done via JSON:API.  You should be able to use `?sort=name`.